tests/data/cases/preview_concatenated_list_comprehensions.py

```diff
@@ -0,0 +1,27 @@
+# flags: --preview --line-length=120
```

Comment:
Can you add a test for short comprehensions that should stay on one line?

How does this look with a lower line length? Is it still an improvement?